tls/s2n_client_key_exchange.c

```diff
@@ -175,6 +193,20 @@ int s2n_rsa_client_key_send(struct s2n_connection *conn, struct s2n_blob *shared
     return 0;
 }
 
+int s2n_kem_client_send_key(struct s2n_connection *conn, struct s2n_blob *shared_key)
+{
+    struct s2n_stuffer *out = &conn->handshake.io;
+
+    struct s2n_blob ciphertext = {0};
```

Comment:
You can use the DEFER_CLEANUP pattern here since there are GUARD macros following. See https://github.com/awslabs/s2n/issues/737